crates/cli/src/introspection/type_unification.rs

```diff
@@ -174,6 +174,16 @@ pub fn unify_object_types(
     merged_type_map.into_values().collect()
 }
 
+/// True iff we consider a to be a supertype of b.
```

Comment:
Super minor spelling `iff`

Reply:
"iff" is an abbreviation for "if and only if"